Library/Formula/mercurial.rb

```diff
@@ -13,6 +13,8 @@ class Mercurial < Formula
     sha256 "53036c632fdfd35d7ccdb85756660e8eeb01437afa6ce68ea35291361bf941ea" => :mavericks
   end
 
+  depends_on "python" if OS.linux?
+
```

Comment:
Please post `brew gist-logs mercurial` without this patch applied to report the error message.
Use instead
```ruby
depends_on "python" unless OS.mac?
```